src/main/java/gift/config/JwtProvider.java

```diff
@@ -0,0 +1,25 @@
+package gift.config;
+
+import gift.entity.Member;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class JwtProvider {
+
+
+    private final String secretKey = "Yn2kjibddFAWtnPJ2AFlL8WXmohJMCvigQggaEypa5E=";
```

Comment:
```suggestion
    @Value("${jwt.secret-key}")
    private String secretKey;
```

환경변수로 키를 관리해보는건 어떨까요?